src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+.nav {
+  &__list {
+    display: flex;
+    flex-direction: row;
+
+    @include fonts-parametrs(12px, 500, 14px, $color-black);
+    
+    text-transform: uppercase;
+  }
+
+  &__item {
+    display: inline-block;
+
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+
+    padding: 23px 0;
+
+    color: $color-black;
```

Comment:
Link color shouldn't be black, it should be **#000e36** as I remember.

Reply:
in this case should be black, for another cases i have variable $color-close-to-black with #000e35 value